pkg/controllers/securitypolicy_associated_controller.go

```diff
@@ -0,0 +1,246 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
```

Comment:
The file name looks weird to me, I did not find a controller defined here

Reply:
I think it can also be called a controller in terms of business functions, because the main function of this file is to retrigger the logic of reconciliation of security policy as soon as it is found that there is a change in the resources associated with the security policy.

Reply:
I think the file structure style with `xxx_associated_controller.go` is not good for our future expansion, because if we think that security policy is the core, and pod and namespace are satellites, then multiple cores may have the same satellites, which will result in the intersection of resource types. E.g. In the future, if we also implement the K8s network policy in NSX operator, we also need to "retrigger the logic of reconciliation of network policy as soon as it is found that there is a change in the resources associated with the network policy". If we also added the `networkpolicy_associated_controller.go`, we should also add the duplicate codes for the duplicate resource types pod and namespace in the new associated controller? So I personally prefer to put `EnqueueRequestForNamespace` in namespace controller, put `EnqueueRequestForPod` in pod controller and the other logic about workflow to securitypolicy_controller.go.

Reply:
ok, it makes sense, let me think about how to rename it.

Reply:
Done, pls re-check